pentoo/rfhs-rfctf/rfhs-rfctf-2021.0.ebuild

```diff
@@ -13,6 +13,7 @@ IUSE="+gui +wifi"
 
 PDEPEND="
 		app-admin/supervisor
+    app-misc/tmux
```

Comment:
mssing tab here?

Reply:
Mmmm, probably... or more likely it was converted to spaces because that's what my editor does... lemme fix that up.